components/config/src/config/mod.rs

```diff
@@ -62,6 +62,10 @@ pub struct Config {
     pub compile_sass: bool,
     /// Whether to minify the html output
     pub minify_html: bool,
+	/// Whether to optimize png files
+	pub optimize_png: bool,
+	/// level to optimize png files if applicable
+	pub optimize_png_level: u8,
```

Comment:
I believe this field can be an `Option<u8>` and you remove the need for `optimize_png`